test-network-function/lifecycle/suite.go

```diff
@@ -113,15 +113,16 @@ var _ = ginkgo.Describe(common.LifecycleTestKey, func() {
 			testPodsRecreation(env)
 
 			testScaling(env)
+			testStateFullSetScaling(env)
 		}
 
 		testOwner(env)
 	}
 })
 
-func waitForAllDeploymentsReady(namespace string, timeout, pollingPeriod time.Duration) { //nolint:unparam // it is fine to use always the same value for timeout
+func waitForAllDeploymentsReady(namespace string, timeout, pollingPeriod time.Duration, podsettype string) { //nolint:unparam // it is fine to use always the same value for timeout
```

Comment:
I'm also changing this part of the code with https://github.com/test-network-function/test-network-function/pull/502 . Depending on which PR gets merged first, we will have to re-do the other.